### ISSUE TITLE
ReadiumCSS: currently here in codebase, can be dependency? (NPM or GitHub)

### ISSUE THREAD

**Issue by danielweck** [edrlab/r2-navigator-js#16]
https://github.com/edrlab/r2-navigator-js/tree/develop/misc/ReadiumCSS

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/r2-streamer-js/issues/30

**Comment by JayPanoz** [comment]
Yeah that would be a good idea as eventually, this is how Readium CSS should be used. 

I’d advocate having a strong feedback loop there if any issue arise (even the tiny ones) and/or feature/change would help so that we can improve Readium CSS accordingly.
